packages/service-library/src/servicelib/aiohttp/long_running_tasks/_dependencies.py

```diff
@@ -1,8 +1,21 @@
+from typing import Any
+
 from aiohttp import web
 
 from ...long_running_tasks._task import TasksManager
-from ._constants import APP_LONG_RUNNING_TASKS_MANAGER_KEY
+from ._constants import (
+    APP_LONG_RUNNING_TASKS_MANAGER_KEY,
+    RQT_LONG_RUNNING_TASKS_CONTEXT_KEY,
+)
 
 
 def get_tasks_manager(app: web.Application) -> TasksManager:
     return app[APP_LONG_RUNNING_TASKS_MANAGER_KEY]
+
+
+def get_task_context(request: web.Request) -> dict[str, Any]:
```

Comment:
STYLE: the word ``task`` might refer to different concepts. I suggest that we denote these as ``long_running_tasks`` or ``long_tasks`` or ``lgr_tasks`` ... or something similar.

Reply:
is this not a bit exagerated? this lives in a module named long_running_tasks already. and everything in here talks about tasks. I do agree with that fact. 
I will think of it for a different PR.
I could also think of renaming the module package into something like rest_async_task instead of long running... since what is long?